pkg/installmanager/installmanager.go

```diff
@@ -232,12 +250,20 @@ func (m *InstallManager) Run() error {
 	cmd := exec.Command("cp", m.InstallConfigMountPath, destInstallConfigPath)
 	err = cmd.Run()
 	if err != nil {
-		log.WithError(err).Errorf("error copying install-config.yaml from %s to %s",
+		m.log.WithError(err).Errorf("error copying install-config.yaml from %s to %s",
```

Comment:
I ran into this same deficiency elsewhere. It is terribly unfortunate that logrus has the same bare functions that can be called outside of an explicit logger.